float32[] laser_readings
float32 goal_distance
float32 goal_angle
